### PR TITLE
Deprecate old clickhouse tables

### PR DESCRIPTION
**Pull request by colegottdank**
| <a href="https://ellipsis.dev" target="_blank">Ellipsis</a> | :rocket: This PR description was created by [Ellipsis](https://www.ellipsis.dev) for commit ff1f4e7116c0651b79d6510f7a1f3677d83d0f5d.  | 
|--------|--------|

### Summary:
This PR deprecates old clickhouse tables and updates the related codebase to remove references to these tables.

**Key points**:
- Deprecated old clickhouse tables `response_copy_v1`, `response_copy_v2`, `response_copy_v3`, `properties_copy_v1`, and `properties_copy_v2`.
- Removed references to these tables from `filterDefs.ts`, `filters.ts`, and `frontendFilterDefs.ts`.
- Updated `logInClickhouse` function in `clickhouseLog.ts` to remove logging related to the deprecated tables.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

**Comment by colegottdank** [comment]
Screenshot 2024-03-01 at 12 51 09 PM
Screenshot 2024-03-01 at 12 51 48 PM
Screenshot 2024-03-01 at 12 51 38 PM
Screenshot 2024-03-01 at 12 51 32 PM
Screenshot 2024-03-01 at 12 51 17 PM

**Comment by colegottdank** [comment]
How to check:

SELECT query
FROM system.query_log
WHERE query LIKE '%response_copy_v3%'
AND query NOT LIKE '%INSERT INTO%'
AND type = 'QueryFinish'
AND event_time >= now() - INTERVAL 5 HOUR
ORDER BY event_time DESC;

**Comment by chitalian** [comment]
@ellipse-dev review this please or somethign

**Comment by chitalian** [comment]
@ellipsis-dev review this please or something

**Comment by colegottdank** [comment]
https://github.com/ellipsis-dev review this please again

**Comment by chitalian** [comment]
This is so satisfying 
image

File: valhalla/jawn/src/lib/shared/filters/filters.ts [hunk 1]
```diff
@@ -229,6 +187,7 @@ const whereKeyMappings: KeyMappings = {
     property_value: "property_with_response_v1.property_value",
     request_created_at: "property_with_response_v1.request_created_at",
     organization_id: "property_with_response_v1.organization_id",
+    threat: "property_with_response_v1.threat",
```
Comment (chitalian) [inline valhalla/jawn/src/lib/shared/filters/filters.ts:190]: Dope

File: worker/src/lib/HeliconeProxyRequest/forwarder.ts [hunk 2]
```diff
@@ -91,9 +91,8 @@ export async function proxyForwarder(
         env
       );
 
+      proxyRequest.threat = threat;
```
Comment (chitalian) [inline worker/src/lib/HeliconeProxyRequest/forwarder.ts:94]: hmmm. is threat ever null?

**Review by chitalian — APPROVED** [review]

**Pull request merged**